test/test_transforms_v2_refactored.py

```diff
@@ -308,36 +308,26 @@ def wrapper(input, *args, **kwargs):
     return wrapper
 
 
-def make_input(input_type, *, dtype=None, device="cpu", spatial_size=(17, 11), mask_type="segmentation", **kwargs):
+def make_input(input_type, *, spatial_size=(17, 11), mask_type="segmentation", **kwargs):
     if input_type in {torch.Tensor, PIL.Image.Image, datapoints.Image}:
-        input = make_image(size=spatial_size, dtype=dtype or torch.uint8, device=device, **kwargs)
+        input = make_image(spatial_size=spatial_size, **kwargs)
         if input_type is torch.Tensor:
             input = input.as_subclass(torch.Tensor)
         elif input_type is PIL.Image.Image:
             input = F.to_image_pil(input)
     elif input_type is datapoints.BoundingBox:
-        kwargs.setdefault("format", datapoints.BoundingBoxFormat.XYXY)
-        input = make_bounding_box(
-            dtype=dtype or torch.float32,
-            device=device,
-            spatial_size=spatial_size,
-            **kwargs,
-        )
+        input = make_bounding_box(spatial_size=spatial_size)
     elif input_type is datapoints.Mask:
-        if mask_type == "segmentation":
-            make_mask = make_segmentation_mask
-            default_dtype = torch.uint8
-        elif mask_type == "detection":
-            make_mask = make_detection_mask
-            default_dtype = torch.bool
-        else:
-            raise ValueError(f"`mask_type` can be `'segmentation'` or `'detection'`, but got {mask_type}.")
-        input = make_mask(size=spatial_size, dtype=dtype or default_dtype, device=device, **kwargs)
+        make_mask = {
+            "segmentation": make_segmentation_mask,
+            "detection": make_detection_mask,
+        }[mask_type]
+        input = make_mask(spatial_size, **kwargs)
```

Comment:
Following up on comment just above, here we're not even leveraging the `mask_type` parameter of `make_input`, which seems to indicate we don't really need `make_input`

Reply:
I don't understand. We are using `mask_type` to select the right function and store that as `make_mask`. What do you mean by 

> here we're not even leveraging the `mask_type` parameter

?

Reply:
> of make_input

Reply:
sorry for my confusion, I didn't realize we were actuall in `make_input`